### DISCUSSION TITLE
Please tell me what you have watched recently! 🎥

### DISCUSSION THREAD

**Discussion by AngieRiera** [NAIST-SE/AttractingContributors#1044] [2024 Attraction Contributions]
I want you to tell what you have watched recently! It can be anything: an anime, movie, a tv series, etc; doesn't have to be anything new either!

Please edit the ReadMe file and send a pull request.
Link [here](https://github.com/AngieRiera/SeriesWatched)

Edit: I have enough contributions now but feel free to keep using this discussion to talk about things you've watched!

**Comment by Drobotsan** [comment]
Hi, I contribute to your project.
Please accept my pull request.

Would you be kind to contribute with my project :D
https://github.com/Drobotsan/Places-in-the-world

Discussion link: https://github.com/NAIST-SE/AttractingContributors/discussions/1140

**Comment by polarbearpppp** [comment]
Hi, I'm contribute to your project , Would you mind to contribute my project 
https://github.com/polarbearpppp/AttractingContributors-Games
